### PR TITLE
Add popup window

### PR DESCRIPTION
**Pull request by Mike47ip**
Merge Popup Window to  Main

### CODE REVIEW

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: - [ ] your popup on mobile device is currently not consistent with the provided figma template, there is no image and the alignment is not consistent. please i kindly suggest you design your mobile popup tomatch the figma template :+1: :sparkles: 
- figma design
inc
- your design
incc

Comment (Efiamotu-1) [inline index.html:1]: - [ ] Your popup for mobile screens doesn't match correctly the provided figma template below, you do not have the appropriate image displayed for each popup and the close button is also not the same. please kindly adapt your popup design to match the provided figma template
- [ ] Also the placement of the `see live & see source` button is not correct as shown on figma please adjust your design correctly. :+1: :sparkles: 
- figma design
but
- your design
buto

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Mike47ip , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mike47ip** [comment]
**### Please ignore Webhint Error. Technical Support say it is a general bug.**

ERR REQUIRE ESM

File: index.html
Comment (usorfaitheloho) [reply index.html:1]: You did well in making the changes suggestedd  by the previous reviewer, please kindly fix the alignment issue in your  buttons for the mobile-popup, as it is always necessary that your design meets the figma template requirement 😄 

Figma template :
image

Your design :
image

Comment (usorfaitheloho) [reply index.html:1]: As suggested by the previous reviewer, kindly fix the alignment of the buttons as they seem to be away from the right positions and also the buttons should be aligned properly and it has a dark background. please implement these changes so your design looks better 😄 

Your design :
image

image

Figma design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
Hi @Mike47ip  :thumbsup:  ,

This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are still some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good job on implementing the design of the previous reviewer ✔️ 
-  Well-structured files ✔️
<br/>
<br/>
<br/>

 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — COMMENTED** [review]
Hi Michael,
 
 While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
 
 ## Invalid Code Review Request
You have submitted this project under the wrong activity name. Kindly ensure to submit this project under the right activity.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html
Comment (anagudelogu) [reply index.html:1]: - [ ] As requested by the previous reviewer, please consider aligning the buttons correctly. Please consider fixing it! 🤓 🔧  Currently, this is how they look like:

|Template|Your design|
|---|---|
| image | image |


- [ ] Also, the close button is supposed to overlay the image. Please consider fixing this! 🤓 🔧 

|Template|Your design|
|---|---|
| image | image |

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @Mike47ip 👋🏼 
I'm Andres.

You've done a great job implementing some of the requested changes for this milestone 🚀, but there are still some which haven't been addressed yet. You're getting closer, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 


 ## Required Changes ♻️
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbelG101 — COMMENTED** [review]
# Hi @Mike47ip 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Please take note that since you're repeating the week with unapproved PR and this is a pair-programming activity you'll have to create a new PR with the commits of your newly assigned coding partner, according to Microverse's policy.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Ofarouq310 — COMMENTED** [review]
Hi @Mike47ip, ✋ 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## 🔴 Invalid Code Review Request

- If you received a message from Student Success about your partner's situation, kindly add a screenshot in the Pull request summary so we can proceed to review your request. Otherwise, if the one informed you is your partner, you need to contact the student success and they will help you.

- Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Mike47ip** [comment]
Screenshot (129)


Please Note that I received a message from Student's success informing me about my partner dropping off. 
Please go on to approve this PR because I have no partner now to commit on my repo. Thank you!

File: index.js [hunk 1]
```diff
@@ -1,20 +1,258 @@
...
-  vector.style.display = "block";
-});
+const vector = document.querySelector('.vector');
+const popwrap = document.querySelector('.Popwrapper');
+const xmenu = document.querySelector('.Xmenu');
+
+vector.addEventListener('click', () => {
+  // vector.style.display = "none";
+  popwrap.style.left = '0';
+});
```
Comment (Ofarouq310) [inline index.js]: - [x] For better clean code, kindly delete any commented code.

File: index.js [hunk 2]
```diff
@@ -1,20 +1,258 @@
...
+        windowpopup.technologies.forEach((tech) => {
+          const techlist = document.createElement('li');
+          techlist.innerText = tech;
+          popupServer.appendChild(techlist);
+        });
+        popupLorem.innerHTML = windowpopup.paragraph;
+        live.href = windowpopup.button1;
+        source.href = windowpopup.button2;
+      }
+    });
```
Comment (Ofarouq310) [inline index.js:205]: - [x] KIndly make sure to style the buttons as much as close to the design, spacings, margins, paddings, and icons, kindly address this issue.

Mobile view

| Your design | Actual design |
| ------------- | ------------- |
| Screenshot 2023-07-01 002159  | Screenshot 2023-07-01 002210 |

Desktop 

| Your design | Actual design |
| ------------- | ------------- |
| Screenshot 2023-07-01 002236 | Screenshot 2023-07-01 002243 |

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @Mike47ip, ✋  

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥈 

giphy

## Highlights
✅ Descriptive PR title & summary.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mike47ip** [comment]
This work has been reviewed long enough. Can you kindly approve it so it so we can all go to sleep?

**Review by shahadat3669 — APPROVED** [review]
Hi @Mike47ip , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Good PR Description
✅ Good PR title
✅ Good use of GitHub Flows
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by cloud007bond** [comment]
- This is an excellent project no error
- No error

**Pull request merged**